README.md

```diff
@@ -1,22 +1,21 @@
-# DSC Devsprints Jan 2020
-## Date and Venue - TBD
-
+DSC Devsprints Jan 2020
+Date and Venue - TBD
 Has contributing to Open source projects been one of those plans for you that just remained a plan? Or did you try contributing to an Open source project but got lost somewhere in the middle due to lack of guidance? Well, Devsprints are here to save the day! During Devsprints, Mentors/Contributors of Open source projects guide participants through everything, from generating the development build to submitting a patch for a bug or improve the project's documentation.
 
-We will also be conducting a brief workshop on Git.
-Links we will refer to:
-https://www.digitalocean.com/community/tutorials/how-to-use-git-a-reference-guide
+We will also be conducting a brief workshop on Git. Links we will refer to: https://www.digitalocean.com/community/tutorials/how-to-use-git-a-reference-guide
 
 https://rogerdudler.github.io/git-guide/
 
-## Agenda
-Prerequisites: Please come with Git already installed.
-TBD
+Agenda
+Prerequisites: Please come with Git already installed. TBD
+
+Suggested Organizations/Projects (in no particular order):
+Organization Name	Website	Contributing Guide
+Pandas	https://github.com/pandas-dev/pandas	https://pandas-docs.github.io/pandas-docs-travis/development/contributing.html
+Matplotlib	https://matplotlib.org/	https://matplotlib.org/devdocs/devel/contributing.html
+Django	https://www.djangoproject.com/	https://docs.djangoproject.com/en/dev/internals/contributing/ https://code.djangoproject.com/wiki/SummerOfCode2020
 
-## Suggested Organizations/Projects (in no particular order):
+# GSOC 2020
 
-| Organization Name | Website                              | Contributing Guide                                                                                                   |
-|-------------------|--------------------------------------|----------------------------------------------------------------------------------------------------------------------|
-| Pandas            | https://github.com/pandas-dev/pandas | https://pandas-docs.github.io/pandas-docs-travis/development/contributing.html                                       |
-| Matplotlib        | https://matplotlib.org/              | https://matplotlib.org/devdocs/devel/contributing.html                                                               |
-| Django            | https://www.djangoproject.com/       |  https://docs.djangoproject.com/en/dev/internals/contributing/  https://code.djangoproject.com/wiki/SummerOfCode2020 |
+SYMPY:-
```

Comment:
Add SymPy to the markdown table present above. Find out the contributing links from their website and add it there.